plugins/org.eclipse.n4js.transpiler.es/src/org/eclipse/n4js/transpiler/es/transform/ModuleWrappingTransformation.xtend

```diff
@@ -740,4 +761,51 @@ class ModuleWrappingTransformation extends Transformation {
 		state.tracer.copyTrace(binding, assignmentStmnt);
 		return assignmentStmnt;
 	}
+
+	protected def ParameterizedCallExpression createExportExpression(SymbolTableEntry entry, Expression expression) {
```

Comment:
since this is N4IDL specific and we keep the separate N4IDL transpiler (for now), it should be moved to the N4IDL transpiler